cuML/src/glm/qn/glm_base.h

```diff
@@ -31,7 +31,7 @@ namespace GLM {
 
 template <typename T>
 inline void linearFwd(SimpleMat<T> &Z, const SimpleMat<T> &X,
-                      const SimpleMat<T> &W, const cublasHandle_t &cublas,
+                      const SimpleMat<T> &W, const cumlHandle_impl &cuml,
```

Comment:
So far we used to pass in `cumlHandle_impl` has the first argument to every function taking it. I think it would be good to stick with that. As name mostly `handle` has used. This also applies to other functions in this PR I will not repeat the comment.